resty

```diff
@@ -168,6 +170,17 @@ for my $var (sort keys %ENV) {
     $env_list .= "env $var;\n";
 }
 
+my $main_include_directive = '';
+my $http_include_directive = '';
+
+if ($main_include) {
+    $main_include_directive = "include $main_include;"
```

Comment:
@agentzh I added an additional commit that does this, so please have another look when you have time.